### PR TITLE
Added the Philips Vereos to the list of scanners

### PR DESCRIPTION
**Pull request by txusser**

### CODE REVIEW

**Comment by NikEfth** [comment]
Are you sure you would like to see all the scanners int he public STIR ? You could hold on to them for now and do this after you have your papers out

**Comment by txusser** [comment]
Well, the information needed to characterize the model is rather public:
https://philipsproductcontent.blob.core.windows.net/assets/20170523/360753349c5d4a6aa46ba77c015e75b4.pdf
No real point on hiding it. Anyway, no hurry on pulling it now.

File: src/recon_test/recontest.cxx [hunk 1]
```diff
@@ -84,7 +84,7 @@ int main(int argc, const char *argv[])
     {
         t.stop();
         std::cout << "Total Wall clock time: " << t.value() << " seconds" << std::endl;
-        return EXIT_SUCCESS;
+        return Succeeded::yes;
```
Comment (KrisThielemans) [inline src/recon_test/recontest.cxx:87]: this is incorrect. revert

File: src/recon_buildblock/IterativeReconstruction.cxx [hunk 2]
```diff
@@ -400,16 +400,16 @@ reconstruct()
 {
   this->start_timers();
 
-  this->target_data_sptr.reset(this->get_initial_data_ptr());
-  if (this->set_up(this->target_data_sptr) == Succeeded::no)
+  shared_ptr<TargetT > target_data_sptr(this->get_initial_data_ptr());
+  if (this->set_up(target_data_sptr) == Succeeded::no)
```
Comment (KrisThielemans) [inline src/recon_buildblock/IterativeReconstruction.cxx:404]: I see no reason for this. It doesn't belong in this PR anyway. Please revert

File: src/recon_buildblock/IterativeReconstruction.cxx [hunk 3]
```diff
@@ -400,16 +400,16 @@ reconstruct()
...
     {
       this->stop_timers();
       return Succeeded::no;
     }
 
   this->stop_timers();
 
-  return this->reconstruct(this->target_data_sptr);
+  return this->reconstruct(target_data_sptr);
```
Comment (KrisThielemans) [inline src/recon_buildblock/IterativeReconstruction.cxx:412]: and revert this

**Review by KrisThielemans — CHANGES_REQUESTED** [review]
could you make this into a PR for just the Vereos? Then I'll squash merge

**Comment by KrisThielemans** [comment]
@txusser any chance you can finalise this?

**Comment by txusser** [comment]
Hi Kris,

This project is currently on hold and rather in the hands of Pablo, but I do not expect that they may continue this in the short term unless someone new incorporates into the team. I still collaborate in the SimPET project but not into coding as I moved to a different institution in January.

**Comment by KrisThielemans** [comment]
sorry to hear that, but do you want to edit this PR a tiny bit such that I can squash-merge it?

**Comment by txusser** [comment]
Maybe @NikEfth is more suitable to do this. My knowledge of C is very limited...

**Comment by KrisThielemans** [comment]
I've added these lines to #59, although disabled, so will close this as no activity.

**Pull request closed without merging**